### PR TITLE
Extend sensor map

### PR DESCRIPTION
**Pull request by OlivierZal**

### CODE REVIEW

**Comment by OlivierZal** [comment]
Hi @robertklep,

Merry Christmas 🎄!
And thanks again for HomeKitty 😄

Retrieves HomeyKit's sensor mapping: https://github.com/swttt/com.swttt.homekit/blob/master/app.js#L237

Needed for [Home Connect](https://homey.app/fr-fr/app/com.home-connect/Bosch-Siemens-Home-Connect/) washer and dryer's open door detection.

**Comment by robertklep** [comment]
I haven't added `other` to most sensor maps because I wanted to find out if it was still necessary, but apparently it is 😅 Thanks for your contribution!

File: lib/maps/contact-sensor.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module.exports = (Mapper, Service, Characteristic) => ({
-  class : 'sensor',
+  class : ['sensor', 'other'],
```
Comment (OlivierZal) [inline lib/maps/contact-sensor.js:2]: https://github.com/swttt/com.swttt.homekit/blob/27ac6e5060a0de1aeb0d4c4e57a0ddaa5bc6b8bf/app.js#L237

**Comment by robertklep** [comment]
Part of v1.2.8 (current test version)

**Pull request merged**